### DISCUSSION TITLE
List supported pens

### DISCUSSION THREAD

**Discussion by zh54tg234f** [flxzt/rnote#440] [General]
My HP Pen does not work with the remapping of button functions. 
So what Stylus Pen to buy?  Which are supported?
Would be nice to have a community list of confirmed working and not yet supported pens.

**Comment by D0V4HKIIN** [comment]
I have an HP Pen too and it works fine.

It is a bit tricky because one of the buttons on my pen is a eraser button which means that it acts as if it was a different input device when pressed down. In that case what you can try is to draw a bit while having the button pressed down, keep it pressed while lifting the pen up. Then click on the eraser (or whatever tool you want to be assigned to that button).

The other button on my pen is a normal button but that one only works when i press it and it's in range to be picked up by the screen but not actually pressed down.

It's not the first time i see people wondering why their buttons don't work and I think rnote should do a better job of making this more clear in the ui.

**Comment by zh54tg234f** [comment]
https://www.amazon.com/s?k=hp+pen&ref=nb_sb_noss

Would you be so kind and tell me the product name since there is more than just one option available.
for my pen i can ensure you it does not work with the button remapping. 

>The other button on my pen is a normal button but that one only works when i press it and it's in range to be picked up by the screen but not actually pressed down.

i don't get what you want to say with the last part.  i assume you mean you press the button without the pen touching the screen just being close to it , is that it?

**Comment by D0V4HKIIN** [comment]
> i assume you mean you press the button without the pen touching the screen just being close to it , is that it?

yes

**Comment by D0V4HKIIN** [comment]
but what i meant is that your pen probably works, it's just rnote's ui that is misleading

**Comment by D0V4HKIIN** [comment]
does your pens button work on other apps like xournalpp?

**Comment by D0V4HKIIN** [comment]
My pen is an HP Pro pen. https://www.hp.com/us-en/shop/pdp/hp-pro-pen-g1

**Comment by zh54tg234f** [comment]
No i tried it  (xournalpp didnt tried)
my pen is 3+ years old model 

Thank you for the information!

**Comment by flxzt** [comment]
I'll just move this into the github discussions section. 

Personally I am using the ThinkPad Active-Pen Pro (but it is jittery, so I wouldn't recommend it), or the small one that comes with the x1-yoga 3rd gen convertible. Both work, but one of the buttons is mapped as a Eraser, so as @D0V4HKIIN mentioned the functionality works a little different and might be confusing: If one of the buttons appears to not work, try holding the button and hovering over the canvas area and then select a tool while still pressing and hovering. When you release it, it should then go back to the previously selected tool and should remember it whenever you press the button again.

**Comment by zh54tg234f** [comment]
Still i think it would be handy for new users to check out which pens works and people are satisfied with.
If we could link this discussion or better a organized list of pens i assume it would help others to have a better experience with the application.  So thanks for sharing your personal experience with the pens you tried.

**Comment by flxzt** [comment]
Absolutely, sounds good! I'd be happy to link a list in the readme and / or on the project website.

**Comment by zh54tg234f** [comment]
I suggest making a Supported-Pens.md within the project and link to it within the README.md so people can add pens via pull requests.
